### PR TITLE
Fix broken subpalette toggling

### LINKED ISSUE #4900: Broken subpalette toggling in 5.x
### Affected version(s)

5.x

### Description

Toggling subpalettes currently seems to be broken in `5.x`:

image

(same problem in `tl_content` when editing content elements, for example)

I didn't get to the bottom of this yet, but the hint might be, that the whole palette is returned in the response, instead of requested subpalette, see screenshots below:

`5.x`:

image

`4.13`:

image

The problem appears on a clean fresh install of `5.x-dev`

### PR DESCRIPTION
**Pull request by ausi**
Fixes #4900

### CODE REVIEW

File: core-bundle/src/Resources/contao/classes/Ajax.php [hunk 1]
```diff
@@ -331,6 +331,7 @@ public function executePostActions(DataContainer $dc)
 						$objVersions->initialize();
 
 						$this->Database->prepare("UPDATE " . $dc->table . " SET " . Input::post('field') . "='" . ((Input::post('state') == 1) ? 1 : 0) . "' WHERE id=?")->execute($this->strAjaxId);
+						DataContainer::clearCurrentRecordCache($this->strAjaxId, $dc->table);
```
Comment (ausi) [inline core-bundle/src/Resources/contao/classes/Ajax.php:334]: Fixes the bug with caching the wrong value.

File: core-bundle/src/Resources/public/core.js [hunk 2]
```diff
@@ -289,10 +289,13 @@ var AjaxRequest =
...
-			if (!el.form.elements.VERSION_NUMBER) {
-				return;
+			var fields = el.form.elements.VERSION_NUMBER || [];
+			if (!fields.forEach) {
+				fields = [fields];
 			}
-			el.form.elements.VERSION_NUMBER.value = /<input\s+[^>]*?name="VERSION_NUMBER"\s+[^>]*?value="([^"]*)"/i.exec(html)[1];
+			fields.forEach(function(field) {
+				field.value = /<input\s+[^>]*?name="VERSION_NUMBER"\s+[^>]*?value="([^"]*)"/i.exec(html)[1];
+			});
```
Comment (ausi) [inline core-bundle/src/Resources/public/core.js:298]: Fixes the bug with a version conflict got shown after enable/disable a subpallete and saving afterwards.

**Review by m-vo — APPROVED** [review]
Code looks good to me. I did not test locally, though.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**